docs/concepts/notifications.md

```diff
@@ -133,13 +133,9 @@ notification = mlrun.model.Notification(
 project.run(..., notifications=[notification])
 ```
 
-For pipelines that have configured notifications, MLRun always sends a `pipeline started` notification. Assuming you have a few notifications 
-configured &mdash; the notification that includes `when=running` determines the parameters for the `pipeline started` notification, for 
+MLRun can also sends a `pipeline started` notification. In order to do so, configure a notification that includes 
```

Comment:
MLRun can also send a `pipeline started` notification. To do that, configure a notification that includes 
`when=running`. The `pipeline started` notification uses its own parameters, for 
example the webhook, credentials, etc., for the notification message.